src/db_util.py

```diff
@@ -99,3 +115,44 @@ def load_pynder_session(username):
 def create_pynder_session(fb_token, fb_id=None):
     pynder_session = pynder.Session(facebook_id=fb_id, facebook_token=fb_token)
     return pynder_session
+
+
+def add_vote(vote):
```

Comment:
dynamic typing-ul din python ne permite sa avem o singura metoda de add() care isi da seama in ce tabel ar trebui bagat obiectul;

+ scapam de duplicate code
- pierdem putin din readability